.azure/pipelines/Phonebook.preview.yml

```diff
@@ -0,0 +1,193 @@
+pr: 
+  branches:
+    include:
+      - master
+
+variables:
+  image_repo: danielhabenicht/phonebook
+  image_tag: pr-$(System.PullRequest.PullRequestNumber) 
```

Comment:
that's maybe duplicate the Tags so it is maybe not the best versioning. (always the case if  the developer update the PR)

Reply:
What do you mean pull request number is always an integer

Reply:
but the build runs after a change on the PR. and a PR can have many changes before merge

Reply:
But the pr number never changes this prs number is 124, nevertheless how often I commit

Reply:
add commit hash